subsys/bluetooth/audio/ascs.c

```diff
@@ -1793,19 +1785,32 @@ static int ase_stream_qos(struct bt_bap_stream *stream, struct bt_audio_codec_qo
 		LOG_WRN("Invalid operation in state: %s", bt_bap_ep_state_str(ep->status.state));
 		*rsp = BT_BAP_ASCS_RSP(BT_BAP_ASCS_RSP_CODE_INVALID_ASE_STATE,
 				       BT_BAP_ASCS_REASON_NONE);
-		return -EBADMSG;
+		return;
+	}
+
+	stream = ep->stream;
+	CHECKIF(stream == NULL) {
```

Comment:
`CHECKIF` should IMO only be used to check for parameters in non-static functions. 

```suggestion
	if (stream == NULL) {
```